PythonVisualizations/Array.py

```diff
@@ -67,8 +77,8 @@ def insert(self, val):
         callEnviron = self.createCallEnvironment(
             self.insertCode.strip(), self.insertCodeSnippets)
 
-        # draw an index pointing to the last cell
-        indexDisplay = self.createIndex(len(self.list), "nItems")
+        # draw an index pointing to the last item in the list
+        indexDisplay = self.createIndex(len(self.list)-1, "nItems", level = -1, color = 'black')
         callEnviron |= set(indexDisplay)

```

Comment:
There's no need to recreate the `nItems` index in each operation.  Use the one built in the `display()` method.